### PR TITLE
Add Shadow Jar-based distro for the default worker process

### PR DESCRIPTION
**Pull request by jimmarino**

### CODE REVIEW

File: distributions/worker-default/build.gradle [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile project(':portability-worker')
+    compile project(':extensions:data-transfer:portability-data-transfer-google')
```
Comment (holachuy) [inline distributions/worker-default/build.gradle]: Do we need to include a cloud extension?

File: build.gradle [hunk 2]
```diff
@@ -61,3 +69,12 @@ configure(sourceProjects()) {
     }
 
 }
+
+def addCloudExtensionDependency(proj) {
+
+    if (proj.rootProject.ext.cloudType == "google") {
+        proj.dependencies { compile project(':extensions:cloud:portability-cloud-google') }
```
Comment (seehamrun) [inline build.gradle]: do we need this case statement? Can we require that a cloud type be set and just use "compile project(':extensions:cloud:portability-cloud-$cloudType')"

Comment (jimmarino) [reply build.gradle]: Good point. I'll add that in

File: build.gradle [hunk 3]
```diff
@@ -36,6 +41,9 @@ plugins {
     id 'java'
 }
 
+ext.cloudType = hasProperty('cloudType') ? cloudType : 'default'
```
Comment (seehamrun) [inline build.gradle]: should we have the default be "local" - I think we talked about moving the cloud / datastore over to its own extension. We already have a local worker too: https://github.com/google/data-transfer-project/tree/master/extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local

**Comment by seehamrun** [comment]
Thanks Jim!

**Pull request merged**